08-cloudwatch-logs/README.md

```diff
@@ -144,9 +158,14 @@ limitations are.*
 
 - What are the minimum and maximum retention times?
 
+> It looks like logs can be stored for 1 day up to indefinately.
+Cloudwatch metrics can be stored for 15 months
```

Comment:
The [PutRetentionPolicy doc](https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutRetentionPolicy.html) takes a value in days, and actually lists a specific set of acceptable values: 1, 3, 5, 7, 14, 30, 60, 90, 120, 150, 180, 365, 400, 545, 731, 1827, and 3653. So you can keep logs there for 10 years.